company/how-to-guides-for-staff/how-to-update-handbook.md

```diff
@@ -29,19 +29,19 @@ After this, click the **Propose file change** button.
 
 ![](../../.gitbook/assets/how-to-update-handbook-commit-changes.png)
 
-Scroll down to compare changes with the original document, request **Reviewers** on the top right navigation, then select **Create pull request**.
+Scroll down to compare changes with the original document, request **Reviewers** on the top right navigation, select relevant labels for the pull request, then select **Create pull request**. 
```

Comment:
I would like to link this out to the code review page for more information about labels and reviewers: https://developers.mattermost.com/contribute/getting-started/code-review/. Thoughts?

Reply:
I noticed that my UI for the PR is different and doesn't have the reviewer and label selection. Is this related to permissions? 
<img width="1078" alt="Screen Shot 2020-02-12 at 20 20 52" src="https://user-images.githubusercontent.com/4223530/74364885-aaf86600-4dd5-11ea-84ff-ff47a5e4a476.png">

Reply:
@justinegeffen Looks like it's related to permissions. Will work with the team to get that resolved..

Reply:
Yes, please link to the labels, thanks!